src/core/bpf/allow_bind/maps.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#pragma once
+
+/* <linux/types.h> must precede <bpf/bpf_helpers.h> due to integer types
+ * in bpf helpers signatures.
+ */
+#include <linux/types.h>
+#include <bpf/bpf_helpers.h>
+#include <linux/bpf.h>
+
+/* Binding is allowed only to ports specified in allow_ports map.
+ * Ports are in host order.
+ * Ports are the same for both v6 and v4 hooks.
+ * max_entries is set from user space with bpf_map__resize
+ * helper.
+ */
+
+struct allow_ports_t {
+        __uint(type, BPF_MAP_TYPE_HASH);
+        __type(key, __u16);
+        __type(value, __u8);
+        __uint(max_entries, 1);
```

Comment:
if you are always specifying max_entries from user-space side, I'd leave max_entries out here, or at least set it to zero. That way it's a bit more obvious, I scratched my head for a bit why just one entry is enough.